CHANGELOG.md

```diff
@@ -12,6 +12,8 @@
 * [#6803](https://github.com/rubocop-hq/rubocop/pull/6803): Fix error for `Style/NumericLiterals` on a literal that contains spaces. ([@pocke][])
 * [#6801](https://github.com/rubocop-hq/rubocop/pull/6801): Fix auto-correction for `Style/Lambda` with no-space argument. ([@pocke][])
 * [#6804](https://github.com/rubocop-hq/rubocop/pull/6804): Fix auto-correction of `Style/NumericLiterals` on numeric literal with exponent. ([@pocke][])
+* Remove argument of `iterator?` and `block_given?`. ([@znz][])
```

Comment:
Thanks for opening a PR.
This change #6714 has not yet been released, so changelog entry is not required.

Reply:
Thanks for your review.
Force updated without changelog entry.